src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx

```diff
@@ -60,12 +60,20 @@ export const DeploySuccess = ({ projectId }: { projectId: number }) => {
       <div className="pt-8 text-4xl font-medium">Congratulations!</div>
       <div className="mt-6 text-xl font-normal">{deployGreeting}</div>
       <div className="flex gap-2 pt-16">
+        <ExternalLink href="https://discord.gg/juicebox">
+          <Button>
+            <Space>
+              <RocketOutlined />
+              <Trans> Share on Discord</Trans>
+            </Space>
+          </Button>
```

Comment:
**suggestion**: Not related to your PR but while we're here, would you mind updating all these buttons to:
- use the `icon` prop
- wrap the `Trans` tag in `span` (for formatting purposes)
- remove any leading space `' '` character from within the `Trans`'s
🙏 🙏 

```suggestion
          <Button icon={<RocketOutlined/>}>
              <span>
                <Trans>Share on Discord</Trans>
              </span>
          </Button>
```